app/uk/gov/hmrc/tariffclassificationfrontend/service/CasesService.scala

```diff
@@ -80,6 +80,14 @@ class CasesService @Inject()(appConfig: AppConfig,
     } yield updated
   }
 
+  def assignCase(original: Case, operator: Operator)
+                  (implicit hc: HeaderCarrier): Future[Case] = {
```

Comment:
formatting

Reply:
What's wrong with it?

Reply:
This follows the pattern of every other method in that class

Reply:
the opening bracket of the implicit param is not parallel with the bracket above